DOC-4264: Update what's new section for 5.2 (#63)
exoplatform/exo-documentation: docs/whatsnew.rst

==================================

Many great enhancements come up in eXo Platform 5.2. Thanks to eXo teams 
and community members who participated by contributing ideas, discussing 
and voting for the new features.

Functional features
~~~~~~~~~~~~~~~~~~~~

:ref:`Spaces Administration <UserGuide.AdministratingSpaces>`

> With the spaces administration feature, it is now possible to define:

Reviewer: ```suggestion
With the spaces administration feature, administrators now have a way to define:
```

- Who can create new spaces
- Who can manage spaces

|image0|

:ref:`Edit activities and comments <Editing-Activities>`

> With eXo Platform 5.2, you can now edit your own activities and comments in the activity stream.

Reviewer: ```suggestion
Users can now edit their own activities and comments in the activity stream.
```

|image1|

:ref:`Empty choice for gender field <About-me>`

> It is possible to choose to not reveal your gender by selecting the empty choice.

Reviewer: ```suggestion
User can now decide to not indicate their gender on their profile.
```

:ref:`Mark all as read in All Notifications page <On-site>`

> Users can mark their all on-site notifications as read with a single click.

Reviewer: ```suggestion
Users can now mark all their on-site notifications as read with a single click.
```

> You can mark your all on-site notifications as read with a single click.

Reviewer: ```suggestion
Users can mark their all on-site notifications as read with a single click.
```

|image2|

:ref:`Number of unread messages in the browser tab <On-site>`

> The number of the unread chat messages is now dynamically updated in the page title  

Reviewer: ```suggestion
The number of unread chat messages is now dynamically added to the page title  
```

> The number of the unread chat messages now appears dynamically in the tab 

Reviewer: ```suggestion
The number of the unread chat messages is now dynamically updated in the page title  
```

> title so you can know there is something requiring your attention even when 

Reviewer: ```suggestion
so that users can easily know when there is something requiring their attention just by
```

> you're working on other browser tabs.

Reviewer: looking at the titles of the browser tabs.

> by looking at the titles of the browser tabs

Reviewer: ```suggestion
by looking at their browser tab
```

|image3|

:ref:`Reset the default avatar or banner <On-site>`

> You can revert your banner or avatar by resetting to the default one.

Reviewer: ```suggestion
Users can now reset their banner or avatar by reverting to the default one.
```

> Users can now reset their banner or avatar by reverting to the default one.

Reviewer: ```suggestion
Users can now reset their profile banner and picture by reverting to the default one.
```

> It is also applicable for spaces.

Reviewer: ```suggestion
It is also applicable for spaces' banners and logos.
```

|image4|

> UX features

Reviewer: ```suggestion
Other user experience improvements
```

~~~~~~~~~~~~

:ref:`Search bar filter in chat <Use-search-bar>`

> When you use the search filter in chat rooms, you can now easily clear your filter

Reviewer: ```suggestion
Clear more easily the search filter in chat rooms, simply
```

by clicking on the cross icon.

|image5|

:ref:`Create event drawer <CreatingNewEvent>`

Creating a new event has been redesigned as an elegant drawer panel for better usability:

|image6|
 
 
:ref:`Badge of new received messages in the mini chat <miniChat>`

> When you scroll down into the mini chat discussion and that your interlocutor 

Reviewer: Keep track of what you've not yet read in mini chat

> sends you new chat messages, a badge with the number of unread messages appears 

Reviewer: thanks to a badge showing the number of unread messages

> :ref:`Badge of new received messages in the mini chat <miniChat>`

Reviewer: ```suggestion
:ref:`Unread messages badge in mini chat <miniChat>`
```

> Keep track of what you've not yet read in mini chat thanks to a badge showing 

Reviewer: ```suggestion
Keep track of unread messages that arrive while you've scrolled up.
```

> the number of unread messages.

Reviewer: ```suggestion

```

|image7|

> In addition to the above UX improvements, we also:

Reviewer: ```suggestion
We also tweaked some UIs :
```

-  Added more spacing at the left and right of the activities for a better readability.

> -  Unified the "Show more" button with the new UX style used accross eXo Platform.

Reviewer: ```suggestion
-  Unified the "Show more" button with the new style used across eXo Platform.
```

.. _TechnicalNovelties:

========================================
Technical novelties in eXo Platform 5.2
========================================

**Security**

> In eXo Platform 5.2 we continued our effort on improving the security by 

Reviewer: ```suggestion
Our continuous effort to improve security lead us to 
```

> changing some permissions and avoiding folder listing through WebDAV.

Reviewer: ```suggestion
change some default permissions to avoid folder listing cases through WebDAV.
```

**Platform RDBMS**

> As in previous versions, we continued to decrease our dependency on JCR 

Reviewer: ```suggestion
We continued to decrease our dependency on JCR 
```

> to store data. Now, the following components rely on the relational 

Reviewer: ```suggestion
for data persistence. Now, the following components rely on the relational 
```

**database** storage for better speed and manageability :

-  Product information
-  Login history
-  Wiki
-  Files
-  Social
-  Notifications
-  Email queue
-  Settings

> You can learn more about our data structure through 

Reviewer: ```suggestion
You can learn more about all data structures in 
```

==================================

Many great enhancements come up in eXo Platform 5.2. Thanks to eXo teams 
and community members who participated by contributing ideas, discussing 
and voting for the new features.

Functional features
~~~~~~~~~~~~~~~~~~~~

:ref:`Spaces Administration <UserGuide.AdministratingSpaces>`

> With the spaces administration feature, it is now possible centrally to define:

Reviewer 2: I would remove "centrally".

Creating a new event has been redesigned as an elegant drawer panel for better usability:

|image6|
 
 
:ref:`Badge of new received messages in the mini chat <miniChat>`

When you scroll down into the mini chat discussion and that your interlocutor 
sends you new chat messages, a badge with the number of unread messages appears 
next to the display name.

|image7|

In addition to the above UX improvements, we also:

-  Added more spacing at the left and right of the activities for a better readability.
-  Unified the "Show more" button with the new UX style used accross eXo Platform.

.. _TechnicalNovelties:

========================================
Technical novelties in eXo Platform 5.2
========================================

**Security**

In eXo Platform 5.2 we continued our effort on improving the security by 
changing some permissions and avoiding folder listing through WebDAV.

**Platform RDBMS**

As in previous versions, we continued to decrease our dependency on JCR 
to store data. Now, the following components rely on the relational 
**database** storage for better speed and manageability :

-  Product information
-  Login history
-  Wiki
-  Files
-  Social

> -  Activities and comments

Reviewer 2: Activities and comments is included in Social.

Author: OK
